src/main/java/aeneas/views/MainView.java

```diff
@@ -221,9 +225,15 @@ void navigateBack() {
     // so only remove and go back if there's multiple things on the stack
     if (paneStack.size() > 1) {
       paneStack.pop();
+      // This is really ugly. Probably to be replaced with an interface for the views or something
+     if(paneStack.peek() instanceof PlaySelectLevelView) {
```

Comment:
That makes the stars get updated when you switch back to the PlaySelectLevelView from the game.